pkgs/development/python-modules/adafruit-platformdetect/default.nix

```diff
@@ -5,12 +5,13 @@
 }:
 
 buildPythonPackage rec {
-  pname = "Adafruit-PlatformDetect";
-  version = "3.4.0";
+  pname = "adafruit-platformdetect";
+  version = "3.4.1";
 
   src = fetchPypi {
-    inherit pname version;
-    sha256 = "sha256-+rZUIZA2P7NZ4jbJsenGlD0OZi5fXFQ/Y5vJo4bmvMo=";
+    pname = "Adafruit-PlatformDetect"
```

Comment:
```suggestion
    pname = "Adafruit-PlatformDetect";
```

Reply:
Yupp, that went wrong with the GitHub editing feature.